.rubocop.yml

```diff
@@ -1,13 +1,51 @@
 inherit_from: .rubocop_todo.yml
 
-# rspec expect{...} is conventional
-Style/SpaceBeforeBlockBraces:
+AllCops:
+  DisplayCopNames: true
+  Include:
+    - Rakefile
+    - lib/**/*.rake
+  Exclude:
+    - Gemfile.lock
+    - '**/*.md'
+    # - bin/**/*
+    - config/**/*.yml
+    # - config/environments/*.rb
+    - script/**/*
+    - vendor/**/*
+
+Bundler/OrderedGems:
+  Exclude:
+    - Gemfile
+
+# --- Layout ---
+
+# expect{...} is conventional
```

Comment:
I'm not sure I believe this.  It's not how they do it in the rspec documentation  (nor anywhere else that I've seen) http://rspec.info/documentation/3.7/rspec-expectations/#Expecting_errors

Reply:
meh - I think it may have been conventional to do expect{...}, even if only in DLSS;  certainly not worth code churn, IMO.